### DISCUSSION TITLE
Another tool for depositing and no excess Bitcoins left on the exchange

### DISCUSSION THREAD

**Discussion by tsims03** [community/community#137136] [Actions]
### Select Topic Area

General

### Body

There is some of us who are having a more difficult time and to have currency to donate or progress in the profession a chance to have free conversion would make us better motivated, more likely to donate and therefore causing a domino effect of efficiency all the way to the exchange and speed would double!

**Comment by Yackens** [comment]
Could you please, rephrase your question? I am having some issues understanding it. Sorry for the inconvenience 😄
